app/javascript/components/admin/manage_users/BannedPendingRow.jsx

```diff
@@ -12,10 +12,12 @@ import {
 } from '@heroicons/react/24/outline';
 import Avatar from '../../users/user/Avatar';
 import useUpdateUserStatus from '../../../hooks/mutations/admin/manage_users/useUpdateUserStatus';
+import { localizeDateTimeString } from '../../../helpers/DateTimeHelper';
 
 export default function BannedPendingRow({ user, pendingTable }) {
   const { t } = useTranslation();
   const updateUserStatus = useUpdateUserStatus();
+  const localizedTime = localizeDateTimeString(user?.created_at, user?.language);
```

Comment:
This should be `currentUser?.language` or else each row will localize the time based on the user in the row's language, not the current user looking at the screen's language